Cucumber/features/project1/feature/find_patient.feature

```diff
@@ -0,0 +1,6 @@
+Feature: Find Patient Feature
+
+  Scenario: Search Scenario
+    Given I search for patient record for Mary
+    Then I see Mary in search results
+    Then I verify table columns are not empty
```

Comment:
Feels like it should be using When instead of Then.
Given I search for patient record for Mary
When I see Mary in search results
Then I verify table columns are not empty